tslint.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Should we create two configs `tsconfig.development.json` and a `tsconfig.production.json` to allow console and debugger on dev builds. And use the production config linting, so console/debugger can only be pushed deliberately.

Reply:
No you shouldn't commit a debugger that is dangerous. If you really want there is a way to skip lint-staged during commit. 

Console throw a warning when running eslint, tslint. A warning doesn't prevent you form committing a file.